packages/cli/src/util/event-listener-to-generator.ts

```diff
@@ -1,15 +0,0 @@
-import { EventEmitter } from 'events';
```

Comment:
File wasn't being used anywhere, and was causing a build error on latest TypeScript.